Run Moonbeam Docker image on Amazon Graviton instances / Apple M1

Hello,

During last years [Amazon WS introduced a lot of ARM based instances](https://aws.amazon.com/ec2/graviton/) and they have a great performance and affordable price. Also, Apple started a movement to the ARM architecture which is actual for Developers. It will be great to have an option to run moonbeam using ARM architecture.

I'm trying to run moonbeam docker image on `c6g.2xlarge - 8 vCPU / 16 RAM` instance but get the error:
```
docker run purestake/moonbeam:v0.14
```
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
standard_init_linux.go:228: exec user process caused: exec format error
```

It seems that Docker provides a way to build images with multi-architecture support - [Leverage multi-CPU architecture support](https://docs.docker.com/desktop/multi-arch/).

It can be helpful to get officially supported images for moonbeam.

Thank you!

Still the same error on [last release](https://github.com/PureStake/moonbeam/releases/tag/v0.17.0)
```bash
   Compiling raw-cpuid v10.2.0
error[E0433]: failed to resolve: could not find `arch` in `self`
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/raw-cpuid-10.2.0/src/lib.rs:80:37
   |
80 |         let result = unsafe { self::arch::__cpuid_count(a, c) };
   |                                     ^^^^ could not find `arch` in `self`

   Compiling wasmi-validation v0.4.0
For more information about this error, try `rustc --explain E0433`.
error: could not compile `raw-cpuid` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/node/perf-test/Cargo.toml
+++ b/node/perf-test/Cargo.toml
@@ -13,9 +13,6 @@ futures = { version = "0.3.1", features = [ "compat" ] }
 hex = "0.4.3"
 libsecp256k1 = { version = "0.5" }
 log = "0.4.8"
-num_cpus = "1.13.0"
-psutil = "3.2.1"
-raw-cpuid = "10.2.0"
 rlp = { version = "0.5" }
 serde = { version = "1.0", features = [ "derive" ] }
 serde_json = "1.0"

